app/store/models/Registration.d.ts

```diff
@@ -74,7 +78,9 @@ export type DetailedRegistration = Pick<
   | 'paymentAmountRefunded'
   | 'LEGACYPhotoConsent'
   | 'photoConsents'
->;
+> & {
+  user: DetailedUser;
```

Comment:
What are you doing here? Why remove the `user` field and then at it back here? What

Reply:
Will I not get a `PublicUser` if I Pick it from the `Registration`?
I need `DetailedUser`s in order to be able to see allergies, right?